### PR TITLE
Fix branch condition in analysis begin step for sonar-secrets-dotnet pipeline

### PR DESCRIPTION
**Pull request by mary-georgiou-sonarsource**

### CODE REVIEW

File: sonar-text-dotnet/src/SonarLint.Secrets.DotNet/Rules/Matching/analysis_begin_step.ps1 [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Set-Location $env:PROJECT_DIR
```
Comment (costin-zaharia-sonarsource) [inline sonar-text-dotnet/src/SonarLint.Secrets.DotNet/Rules/Matching/analysis_begin_step.ps1:1]: This file seems to be now in 2 places. Is this intended?

File: sonar-text-dotnet/pipeline_scripts/analysis_begin_step.ps1 [hunk 2]
```diff
@@ -1,25 +1,25 @@
 Set-Location $env:PROJECT_DIR
 
-if ($env:CIRRUS_BRANCH -eq "master") {
```
Comment (costin-zaharia-sonarsource) [inline sonar-text-dotnet/pipeline_scripts/analysis_begin_step.ps1:3]: I would remove the negation and move the parameters in the other branch.

File: sonar-text-dotnet/src/SonarLint.Secrets.DotNet/Rules/Matching/analysis_begin_step.ps1 (hunk 1 above)
Comment (mary-georgiou-sonarsource) [reply sonar-text-dotnet/src/SonarLint.Secrets.DotNet/Rules/Matching/analysis_begin_step.ps1:1]: Glitch in the matrix.
Fixed!

**Review by costin-zaharia-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**